src/Microsoft.ML.Data/Transforms/FeatureContributionCalculationTransform.cs

```diff
@@ -25,10 +25,29 @@
 namespace Microsoft.ML.Data
 {
     /// <summary>
-    /// The FeatureContributionCalculationTransformer computes model-specific contribution scores for each feature.
+    /// The FeatureContributionCalculationTransformer computes model-specific per-feature contributions to the score of each data point.
     /// See the list of currently supported models below.
     /// </summary>
     /// <remarks>
+    /// <para>
+    /// Scorind a data set with a trained model produces a score, or prediction, for each data sample. To understand and explain these predictions
+    /// it can be useful to inspect which features influenced them most significantly. FeatureContributionCalculationTransformer computes a model-specific
+    /// list of per-feature contributions to the score for each data sample. These contributions can be positive (they make the score higher) or negative
+    /// (they make the score lower).
+    /// </para>
+    /// <para>
+    /// For linear models, the contribution of a given feature is just equal to the product of feature times the corresponding weight. Similarly, for
+    /// Generalized Additive Models (GAM), the contrubution of a feature is equal to the shape function for the given feature evaluated at the feature value.
+    /// </para>
+    /// <para>
```

Comment:
I think it might be best to move these remarks into the documentation for the individual implementations.

Reply:
And keep this section high level on what we mean by `contributions` and how to use them.

Reply:
I added some documentation for the individual implementations, but I also kept it in the main FCC documentation as I was not able to reference the implementations from the FCC doc.